virt/getting_started/virt-web-console-overview.adoc

```diff
@@ -251,6 +251,11 @@ To store Red Hat templates in multiple projects, xref:../../virt/getting_started
 
 Preview features are disabled by default and must not be enabled in production environments.
 
+|*Automatic subscription of new RHEL VirtualMachines* section
+|Expand this section to enable automatic subscription for {op-system-base-full} VirtualMachines.
+
+Cluster administrator permissions are required to enable this feature.
```

Comment:
This is missing info. The cluster admin must provide an organization ID and activation code. It's not like any cluster admin can enable the feature.